connectors/src/connectors/intercom/temporal/client.ts

```diff
@@ -11,12 +11,19 @@ import { getTemporalClient } from "@connectors/lib/temporal";
 import logger from "@connectors/logger/logger";
 import { ConnectorResource } from "@connectors/resources/connector_resource";
 
-export async function launchIntercomSyncWorkflow(
-  connectorId: ModelId,
-  fromTs: number | null,
-  helpCenterIds: string[] = [],
-  teamIds: string[] = []
-): Promise<Result<string, Error>> {
+export async function launchIntercomSyncWorkflow({
```

Comment:
@spolu I tried to pass `forceResync` to the `args` attribute of `await client.workflow.signalWithStart(intercomSyncWorkflow, {})` as we do for other workflow but after making a lot of tests on my local I understand that there is no way to edit the args of a cron based workflow -> you have to stop it and relaunch it with the new args you want. 

Meaning that for a full resync I would have to: 
- stop the workflow for this connector
- launch it with args set will fullSync = true
- stop it when ended and then relaunch the next one with fullSync = false. 

I decided on another approach and propose to use the signalArgs instead. 
I'm passing `forceResync` to the signal args that are then passed to the workflows that are spawned. 
Looks like it's working as expected as is.